.travis.yml

```diff
@@ -26,3 +36,7 @@ script:
     - OMP_NUM_THREADS=1 tests/src/eigenwave3d
     - python tests/eigenwave3d.py default --output
     - python tests/eigenwave3d.py read
+    - python tests/eigenwave3d.py pluto --execute --nthreads=4
+    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
+    - python tests/eigenwave3d.py pluto --output
+    - python tests/eigenwave3d.py read
```

Comment:
done.
Also I've wrote a script for testing tiling, do I add it to pluto ? or do I add it in branch benchmarking

Date: Fri, 11 Sep 2015 08:12:28 -0700
From: notifications@github.com
To: opesci-fd@noreply.github.com
CC: zlcghoul@hotmail.com
Subject: Re: [opesci-fd] Pluto (#45)

In .travis.yml:

> @@ -26,3 +36,7 @@ script:
>      - OMP_NUM_THREADS=1 tests/src/eigenwave3d
>      - python tests/eigenwave3d.py default --output
>      - python tests/eigenwave3d.py read
> +    - python tests/eigenwave3d.py pluto --execute --nthreads=4
> +    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
> +    - python tests/eigenwave3d.py pluto --output
> +    - python tests/eigenwave3d.py read

There's no need for the last one, since it doesn't actually invoke pluto.


―
Reply to this email directly or view it on GitHub.